### ISSUE TITLE
How to manually set learning rate ?

### ISSUE THREAD

**Issue by yashmehta10** [pycaret/pycaret#344]

**Comment by pycaret** [comment]
Just pass learning rate inside create model.

For example:

>>> create_model('lightgbm', learning_rate = 0.3)
